python/paddle/tensor/linalg.py

```diff
@@ -2503,3 +2502,107 @@ def __check_input(x, UPLO):
         attrs={'UPLO': UPLO,
                'is_test': is_test})
     return out_value
+
+
+def lstsq(x, y, rcond=1e-15, driver=None, name=None):
```

Comment:
please add comments

Reply:
The imlementation of GPU kernel and docs will be added in next PR.